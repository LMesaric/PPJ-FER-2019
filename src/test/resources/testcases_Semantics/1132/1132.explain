semantic: use return; only in functions returning void
note: 'return ;' on line 17
