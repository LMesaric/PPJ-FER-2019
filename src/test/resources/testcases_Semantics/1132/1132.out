<naredba_skoka> ::= KR_RETURN(17,return) TOCKAZAREZ(17,;)
